yb-voyager/src/tgtdb/oracle.go

```diff
@@ -562,7 +562,7 @@ func (tdb *TargetOracleDB) importBatch(conn *sql.Conn, batch Batch, args *Import
 	password := tdb.tconf.Password
 	connectString := tdb.getConnectionString(tdb.tconf)
 	oracleConnectionString := fmt.Sprintf("%s@\"%s\"", user, connectString)
-	sqlldrArgs := fmt.Sprintf("userid=%s control=%s log=%s DIRECT=TRUE NO_INDEX_ERRORS=TRUE", oracleConnectionString, sqlldrControlFilePath, sqlldrLogFilePath)
+	sqlldrArgs := fmt.Sprintf("userid=%s control=%s log=%s DIRECT=TRUE NO_INDEX_ERRORS=TRUE SKIP=1 ERRORS=0", oracleConnectionString, sqlldrControlFilePath, sqlldrLogFilePath)
```

Comment:
what do these options mean? can you pls comment here, or link to doc

Reply:
ERRORS=0, set the error limit to 0 which means that any single error in the datafile can't be skipped. `sqlldr` should error out at that time itself. 
SKIP=1 is to skip the first row which is the header.
will add a comment.